src/groupByKey.ts

```diff
@@ -1,7 +1,18 @@
-type GroupsMap<T> = {
-  [key: string]: T[];
-};
-
-export function groupByKey(items, key) {
+export function groupByKey<T>(items: T[], key: keyof T): object {
   // write code here;
+  const result: any = {};
+
+  items.forEach((itemsElement: T) => {
+    const variable: string = String(itemsElement[key]);
```

Comment:
Same with variable. Better specify what variable it is

Reply:
In result, this variable will be string.

Reply:
I rename it)